ste/uploader-blobFS.go

```diff
@@ -138,6 +139,10 @@ func (u *blobFSUploader) RemoteFileExists() (bool, error) {
 func (u *blobFSUploader) Prologue(state common.PrologueState) {
 	jptm := u.jptm
 
+	if flushThreshold == 0 {
+		flushThreshold = int64(u.chunkSize * 7500)
```

Comment:
chunkSize can be different for each file, right? flushThreshold should not be a global variable.